components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.store.ui/src/main/resources/web/store-mediator/edit-mediator.jsp

```diff
@@ -70,9 +79,29 @@
         <td>
             <table class="normal">
                 <tr>
-                    <td><fmt:message key="mediator.store.messageStore"/> <span class="required">*</span>
+                    <td><h3><fmt:message key="mediator.store.messageStore"/> <span class="required">*</span></h3>
+                    </td>
+                </tr>
+
+                <tr>
+                    <td class="leftCol-small">
+                        <fmt:message key="mediator.store.specifyAs"/> :
                     </td>
                     <td>
+                        <input type="radio" id="specify"
+                               onclick="javascript:displayElement('value', true); javascript:displayElement('xpath', false);"
+                               name="specifyAs" <%=!isExpression ? "checked=\"checked\" value=\"Value\"" : "value=\"Value\""%>/>
+                        <fmt:message key="mediator.store.value"/>
+                        <input type="radio"
+                               onclick="javascript:displayElement('value', false); javascript:displayElement('xpath', true);"
+                               name="specifyAs" <%=isExpression ? "checked=\"checked\" value=\"Expression\"" : "value=\"Expression\""%>/>
+                        <fmt:message key="mediator.store.xpath"/>
+                    </td>
+                </tr>
+
+                <tr id="value" <%=isExpression ? "style=\"display:none\";" : ""%>>
```

Comment:
shall we move inline css to a file

Reply:
I had it like this because in all other files, this particular style is given in-line. So that it will be consistent. But if you want, I can move this to an external CSS file.